frontend/src/types/Graph.ts

```diff
@@ -571,12 +571,14 @@ export const NodeAttr = {
   httpInNoResponse: 'httpInNoResponse',
   httpOut: 'httpOut',
   id: 'id',
+  isAmbientNamespace: 'isAmbientNamespace',
```

Comment:
Is there a reason why this isn't also `isAmbient`?

Reply:
I was thinking in a way to differentiate when a Node isAmbient when there is a check in the pod annotation (It includes the ztunnel redirection annotation) or because it is an Ambient namespace. I think as you said in the next comment, it would be better to add it to models.Namespace, when it is a Namespace check